cmsplugin_profile/cms_plugins.py

```diff
@@ -69,10 +69,14 @@ class ProfileGridPromoPlugin(CMSPluginBase):
         ("Featured Profiles", {
             'fields': ('profiles_field', ),
             'description':
-                _(u"{} profiles will be featured in this promo, marked by the"
-                  " border around the thumbnails. Click on a thumbnail to add"
-                  " or remove it from the group of featured profiles."
-                  .format(MAX_PROMO_PROFILES)),
+                _(u"Profile entries can be selected to be featured in a Profile"
```

Comment:
Yes, you are right, this was a mistake.
It should have been something like `_('{nr_profiles} profiles ...').format(MAX_PROMO_PROFILES)`.
I don't know if we should create a different pull request just for this since we don't use the translation engine, but it is a good point and we should watch for it.